src/features/game/lib/processEvent.ts

```diff
@@ -125,10 +125,11 @@ export const maxItems: Inventory = {
   "Love Letter": new Decimal(400),
 
   // Emblems
-  "Goblin Emblem": new Decimal(500),
-  "Bumpkin Emblem": new Decimal(500),
-  "Sunflorian Emblem": new Decimal(500),
-  "Nightshade Emblem": new Decimal(500),
+  "Goblin Emblem": new Decimal(90_000),
```

Comment:
If someone has > 90000 to claim (on Jun 14) what happens? Do they claim in batches or just never claim b/c it's over the limit?